test/metadata.test.js

```diff
@@ -267,6 +267,21 @@ if (TestEnv.isNodeJS) {
     });
   });
 
+  describe("describe value type", function() {
+    it("should return describeValueType", function(done) {
+      conn.metadata.describeValueType('{http://soap.sforce.com/2006/04/metadata}ApexClass', function(err, result) {
+        assert.ok(_.isBoolean(result.apiCreatable));
+        assert.ok(_.isBoolean(result.apiDeletable));
+        assert.ok(_.isBoolean(result.apiReadable));
+        assert.ok(_.isBoolean(result.apiUpdatable));
+        assert.ok(_.isBoolean(result.partialSaveAllowed));
+        assert.ok(_.isBoolean(result.testRequired));
```

Comment:
Doing network calls in unit tests is not usually a good practice. I suggest using a library which can "record" a real response at test write time and "replay" it at runtime. There are multiple such libs available IIRC.

Reply:
Totally agree with your remark. However, all the tests in this module are performing network calls (which shouldn't be done). This change to our fork is taken from another fork which implemented this missing feature together with the test that adheres to the rest of the tests in this library test suites. Let's have this as a tech debt which we will need to deal with. For now I suggest we move on to unblock the missing feature. WDYT?

Reply:
Totally agree with your remark. However, all the tests in this module are performing network calls (which shouldn't be done). This change to our fork is taken from another fork which implemented this missing feature together with the test that adheres to the rest of the tests in this library test suites. Let's have this as a tech debt which we will need to deal with. For now I suggest we move on to unblock the missing feature. WDYT?